backend/compact-connect/lambdas/purchases/data_model/schema/base_record.py

```diff
@@ -0,0 +1,127 @@
+# pylint: disable=invalid-name
+# We diverge from PEP8 variable naming in schema because they map to our API JSON Schema in which,
+# by convention, we use camelCase.
+from abc import ABC
+from datetime import datetime, UTC
+from typing import Type
+
+from marshmallow import Schema, RAISE, EXCLUDE, post_load, pre_dump
+from marshmallow.fields import String, Date, UUID, List
+from marshmallow.validate import Regexp
+
+from exceptions import CCInternalException
+
+
+class StrictSchema(Schema):
+    """
+    Base Schema explicitly stating what we do if unknown fields are included - raise an error
+    """
+    class Meta:
+        unknown = RAISE
+
+
+class ForgivingSchema(Schema):
+    """
+    Base schema that will silently remove any unknown fields that are included
+    """
+    class Meta:
+        unknown = EXCLUDE
+
+
+class SocialSecurityNumber(String):
```

Comment:
You've got a few unused declarations in here:
`SocialSecurityNumber`
`Set`
`ITUTE164PhoneNumber`
`SSNIndexRecordSchema`

We should probably remove them.